### ISSUE TITLE
Disability-ratings calculator flagged for accessibility

### ISSUE THREAD

**Issue by cmaeng** [department-of-veterans-affairs/va.gov-team#28978]
## Instructions

1. Click or press the gear icon next to the "Labels" heading on the right. Search for your team in the labels list. If your team is not listed in the Labels menu, please leave a comment after you have created this issue. The Product Support team will create a new label for your team.
2. If this issue is being opened by a VFS team, please add the `vsp-product support` label.
3. **Issue tickets should be tested when you have created a pull request.** This gives the 508, or quality assurance, person time to review your code before it is merged into the `master` branch. Once code is merged into `master` it will be deployed to Staging. Reviewing and correcting issues becomes more difficult at this point.
4. Text inside `<-- comment -->` blocks will not appear in your issue ticket. These comments offer guidance on what information you should provide for each section.
5. Leave a link here to the 508 defect severity level: 

    - [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Clarence_

## User Story or Problem Statement

The `disability-ratings` calculator react widget on [this page](https://www.va.gov/disability/about-disability-ratings/) is getting flagged by the accessibility scan because of the grey font color used for text = In 10% increments.

## Details

Jenkins log:
`Failed [fail]: (
  Description:  Elements must have sufficient color contrast
  Impact:       serious
  Rule ID:      color-contrast
  URL:          http://content-build:3001/disability/about-disability-ratings/
  DOM node:     
  <span class="vads-u-color--gray-medium vads-u-display--none small-screen:vads-u-display--block">In 10% increments</span>
  .vads-u-color--gray-medium
   More help:    https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI`

[Slack thread for context](https://dsva.slack.com/archives/C01RAS1KAQK/p1629389393023200)

## Acceptance Criteria

- [ ] the font color used in the react widget has enough contrast to pass accessibility scans

## Environment

* Operating System, including `<VERSION>` or "latest": Windows
* Browser, including `<VERSION>` or "latest": Chrome

## Steps to Recreate

1. Enter `<https://www.va.gov/disability/about-disability-ratings/>` in browser
2. Start screenreading device listed in Environment

## Proposed Solution (if known)

Change the color contrast used in the react widget. I think this would go to the Public Websites team.

## Screenshots or Trace Logs

![Screen Shot 2021-08-19 at 12.23.04 PM.png](https://images.zenhubusercontent.com/5e3ad9b2c4b3070f908bf89c/faa3574f-a07b-4e7b-8958-5a0323870c93)
